src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.cs

```diff
@@ -46,6 +68,7 @@ static ExpressionCompiler()
                         var builder = ArrayBuilder<LocalAndMethod>.GetInstance();
                         string typeName;
                         var assembly = context.CompileGetLocals(
+                            () => s_NoAliases,
```

Comment:
If this is the only use, maybe the field should be a GetAlias?  Of course, the point would be moot if we dropped the delegate altogether.